guides/release/routing/specifying-a-routes-model.md

```diff
@@ -86,33 +86,33 @@ method.
 
 Note: A route with a dynamic segment will always have its `model` hook called when it is entered via the URL.
 If the route is entered through a transition (e.g. when using the [link-to](../../templates/links/) Handlebars helper),
```

Comment:
the `<LinkTo />` component

Reply:
@chancancode Looks like the relative link `../../templates/links/` is dead. Could you point me to the correct page? I am having trouble finding it.

Reply:
https://github.com/ember-learn/guides-source/blob/octane/guides/release/templates/links.md

It should work 🤔

Reply:
@chancancode Looks like putting `<LinkTo />` directly into the MD link box won't work. Because of this and that I couldn't find templates/links in the left nav, I assumed the link broke 🤣I had to put it in side back ticks to work.

Reply:
Thank you for checking! :scream: Do you mind checking if the other PRs we merged lately have this problem?

Reply:
@chancancode I checked the recently closed PR's. I can confirm that none of them have this issue.

Reply:
Thank you!!